sigstore-gradle/sigstore-gradle-sign-base-plugin/src/main/kotlin/dev/sigstore/sign/SigstoreSignExtension.kt

```diff
@@ -39,7 +39,7 @@ abstract class SigstoreSignExtension(private val project: Project) {
     abstract val sigstoreJavaVersion : Property<String>
 
     init {
-        sigstoreJavaVersion.convention("0.2.0")
+        sigstoreJavaVersion.convention("0.3.0")
```

Comment:
Nice, so this just works now?

Reply:
It does. Apparently, `0.2.0` misses `BundleFactory`, so I bump "default" `sigstore-java` to `0.3.0` here.